niceday_client/definitions.py

```diff
@@ -1,6 +1,9 @@
-from enum import IntEnum
+from enum import IntEnum, Enum
 
 USER_PROFILE_KEYS = ['firstName', 'lastName', 'location', 'birthDate', 'gender']
 
 class Tracker(IntEnum):
     SMOKING = 1
+
+class TrackerName(Enum):
+    SMOKING = 'tracker_smoking'
```

Comment:
We could also try and combine this? It's of course super confusing because both are actually identifiers.